src/routes/dashboard/components/Dashboard.js

```diff
@@ -1,23 +1,48 @@
 const React = require('react');
 const T = require('prop-types');
-const { PageWrapper } = require('styles/global-components.js');
-const { Typography } = require('@material-ui/core');
+const { PageWrapper, UserListItem } = require('styles/global-components.js');
+const { Typography, List, ListItemText } = require('@material-ui/core');
 
 module.exports = class Dashboard extends React.Component {
 
     static propTypes = {
-        firstName: T.string.isRequired
+        firstName: T.string.isRequired,
+        getUsers: T.func.isRequired,
+        users: T.array.isRequired
     };
 
+    componentDidMount() {
+
+        this.props.getUsers();
+    }
+
     render() {
 
-        const { firstName } = this.props;
+        const { firstName, users } = this.props;
 
         return (
 
             <PageWrapper>
                 <Typography variant='display2' gutterBottom>Dashboard</Typography>
-                <Typography>Welcome to your dashboard, {firstName}!</Typography>
+                <Typography gutterBottom>Welcome to your dashboard, {firstName}!</Typography>
+                <Typography variant='display1'>User List</Typography>
+                <List>
+                    {
+                        users.map( (user) => (
+
+                            <UserListItem
+                                key={user.id}
+                                dense
+                                button
+                            >
+                                <ListItemText
+                                    primary={user.fullName}
+                                    secondary={user.email}
+                                />
```

Comment:
We can also show `created` and `updated` dates if that's of interest. I opted to keep it simple, but we have options!